src/net_processing.cpp

```diff
@@ -803,6 +803,11 @@ bool GetNodeStateStats(NodeId nodeid, CNodeStateStats &stats) {
     return true;
 }
 
+int GetNumberOfPeersWithValidatedDownloads() EXCLUSIVE_LOCKS_REQUIRED(cs_main)
```

Comment:
Annotations in definitions are ignored, they need to be in the declaration

```suggestion
int GetNumberOfPeersWithValidatedDownloads()
```

Reply:
Fixed.